.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+   - '4.5'
+before_script:
+   - npm install -g grunt
+script:
+   - grunt
```

Comment:
Please make this more specific so it's easier to tell what Travis is testing. e.g. `grunt standards`